Description: Do not use intersphinx
 Intersphinx does network access, which is forbidden during build.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-09-25

--- python-muranoclient-0.5.5.orig/doc/source/conf.py
+++ python-muranoclient-0.5.5/doc/source/conf.py
@@ -9,7 +9,6 @@ project = 'python-muranoclient'
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc',
-              'sphinx.ext.intersphinx',
               'sphinx.ext.coverage',
               'sphinx.ext.pngmath',
               'sphinx.ext.ifconfig',
